.meta/sources/journald.toml.erb

```diff
@@ -67,6 +67,28 @@ If empty or not present, all units are accepted. \
 Unit names lacking a `"."` will have `".service"` appended to make them a valid service unit name.\
 """
 
+[sources.journald.options.exclude_units]
+type = "[string]"
+common = true
+default = []
+examples = [["badservice", "sysinit.target"]]
+description = """\
+The list of units names to exclude from monitoring. \
+Unit names lacking a `"."` will have `".service"` appended to make them a valid service unit name.\
+"""
+
+[sources.journald.options.units]
```

Comment:
We typically don’t show deprecated options in the docs, assuming they are still silently reported. I’m in favor or simply removing this from the docs.

Reply:
Of course. Done.